### PR TITLE
update description on validation_split

### PR DESCRIPTION
**Pull request by Krisloveless**
https://github.com/keras-team/keras-preprocessing/blob/6ff9af6971f97f69c39d090be15c2f7d93aa4d6d/keras_preprocessing/image/numpy_array_iterator.py#L90
https://github.com/keras-team/keras-preprocessing/blob/6ff9af6971f97f69c39d090be15c2f7d93aa4d6d/keras_preprocessing/image/numpy_array_iterator.py#L101-L102

According to these 3 lines, the validation set is actually from the first percentage not the last. I personally think this is a very urgent issue to be dealt with since people without coding experience can only rely on the documents for doing project.

### PR Overview

- [ ] This PR requires new unit tests [y/n] (make sure tests are included)
- [ ] This PR requires to update the documentation [y/n] (make sure the docs are up-to-date)
- [ ] This PR is backwards compatible [y/n]
- [ ] This PR changes the current API [y/n] (all API changes need to be approved by fchollet)

### CODE REVIEW

**Comment by fchollet** [comment]
The current description is accurate (you can check the code: https://github.com/keras-team/keras/blob/master/keras/engine/training.py#L1116)
